pages/jobs/new.tsx

```diff
@@ -72,57 +78,73 @@ const NewJob = () => {
   }, [draft]);
 
   return (
-    <div className="container">
-      <FormLabel>
-        <Checkbox
-          onChange={() => setDraft({ ...draft, canContact: !draft.canContact })}
-          isChecked={draft.canContact}
+    <Box maxWidth={"80%"} width="1200" margin="auto">
```

Comment:
```suggestion
    <Box maxWidth="80%" width="1200" margin="auto">
```

Small detail, no big deal